package.json

```diff
@@ -42,9 +42,7 @@
     "async": "2.4.0",
```

Comment:
Can you hard-peg the version numbers? We do that as a best practice in case anyone is still using `npm` so we don't accidentally get breaking minor versions.

Also, I think `ramda` is a true dep, not just a devdep?

Lastly, could you also prune the deps (both kinds) we don't need? For instance, I don't see any references to `async` anywhere.